common/djangoapps/student/models.py

```diff
@@ -1435,3 +1435,18 @@ class DashboardConfiguration(ConfigurationModel):
     @property
     def recent_enrollment_seconds(self):
         return self.recent_enrollment_time_delta
+
+
+class LinkedInUrlConfiguration(ConfigurationModel):
+    """
+    Register Linked Add to Profile Link
+    """
+    # linked-in URL field
+    linkedin_url = models.URLField(
+        blank=True,
+        help_text="A LinkedIn URL for the Add-to-profile Certificates. e.g http://www.linkedin.com/profile/add?_ed="
+                  "0_0dPSPyS070e0HsE9HNz_13_d11_"
```

Comment:
Nitpick: this help text should be internationalized.  You can use `uggettext_lazy` to do this:
```
from django.utils.translation import ugettext_lazy

...
help_text=ugettext_lazy(
     u"A LinkedIn URL for the Add-to-profile Certificates.  "
     u"e.g. http://www.linkedin.com/profile/add?_ed=0_0dPSPyS070e0HsE9HNz_13_d11_"
)
```